### PR TITLE
add github templates

### PR DESCRIPTION
**Pull request by lichunzhu**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Create a topic branch from where you want to base your work. This is usually `master`.
+- Make commits of logical units and add test case if the change fixes a bug or adds new functionality.
+- Run tests and make sure all the tests are passed.
+- Make sure your commit messages are in the proper format (see below).
+- Push your changes to a topic branch in your fork of the repository.
+- Submit a pull request.
+- Your PR must receive LGTMs from two maintainers.
```
[the remark this answers is not in the document]
Comment (lichunzhu) [reply CONTRIBUTING.md]: No, I think we needn't. I have addressed this.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+name: "\U0001F41B Bug Report"
+labels: "bug"
```
Comment (kennytm) [inline .github/ISSUE_TEMPLATE/bug-report.md:3]:
```suggestion
```

currently we don't care about issue type and have no labels for them.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: "\U0001F680 Feature Request"
+labels: "feature-request"
```
Comment (kennytm) [inline .github/ISSUE_TEMPLATE/feature-request.md:3]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+name: "\U0001F914 Question"
+labels: "question"
```
Comment (kennytm) [inline .github/ISSUE_TEMPLATE/question.md:3]:
```suggestion
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+This project contains unit tests and integration tests with coverage collection.
+See [README.md](https://github.com/pingcap/dumpling#building) for how to execute tests.
+
+### Updating dependencies
+
+Dumpling manages dependencies using [Go 1.13 module](https://github.com/golang/go/wiki/Modules).
+To add or update a dependency, either
+
+* Use the `go mod edit` command to change the dependency, or
+* Edit `go.mod` and then run `make update` to update the checksum.
```
Comment (kennytm) [inline CONTRIBUTING.md:35]:
```suggestion
* Edit `go.mod` and then run `go mod tidy` to update the checksum.
```

dumpling doesn't have `make update` command. and in lightning it is equivalent to `go mod verify && go mod tidy` anyway.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Create a topic branch from where you want to base your work. This is usually `master`.
+- Make commits of logical units and add test case if the change fixes a bug or adds new functionality.
+- Run tests and make sure all the tests are passed.
+- Make sure your commit messages are in the proper format (see below).
+- Push your changes to a topic branch in your fork of the repository.
+- Submit a pull request.
+- Your PR must receive LGTMs from one maintainer.
```
Comment (kennytm) [inline CONTRIBUTING.md:47]:
```suggestion
- Your PR must receive an LGTM from one maintainer.
```

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (lichunzhu) [reply .github/ISSUE_TEMPLATE/bug-report.md]: addressed

**Pull request merged**